tcl/ldict.tcl

```diff
@@ -105,3 +105,25 @@ proc qc::ldict2tbody {ldict colnames} {
     return $tbody
 }
 
+proc qc::ldict_mappings_equal {ldict1 ldict2} {
```

Comment:
What would qc::ldict_equal look like ?

Reply:
It would look the same, but with `qc::dicts_equal` instead of `qc::dict_mappings_equal`.
It would be hard for the caller to specify that the dicts need to be sorted before comparison.